src/utils/helpers.js

```diff
@@ -24,6 +24,28 @@ export const getDatesDifference = (date, period) => {
   return diffInDays || 1;
 };
 
+/**
+ * get dates difference function
+ *
+ * @param {String} date date (ex: 2022-01-24T12:47:07.098Z)
+ * @param {Number} period period to compare if the dates difference exceed it
+ *
+ * @return {Number} difference between 2 dates
+ */
+export const getDifferenceBetweenTwoDates = (firstDate, secondDate, period) => {
+  const cleanedFirstDate = firstDate?.toLocaleDateString();
+  const cleanedSecondDate = secondDate?.toLocaleDateString();
+
+  const diffInMs = new Date(cleanedSecondDate) - new Date(cleanedFirstDate);
+  const diffInDays = Math.ceil(diffInMs / (1000 * 60 * 60 * 24) + 1);
```

Comment:
There can be no assurance that the first is less than the second. If we want an absolute value we should replace with 
```suggestion
  const diffInDays = Math.abs(Math.ceil(diffInMs / (1000 * 60 * 60 * 24) + 1));
```